references.bib

```diff
@@ -0,0 +1,9 @@
+@misc{gtsam,
```

Comment:
Can you add the main papers as well? See my publications page, also include the IMU pre-integration papers.

Reply:
I don’t have cycles at the moment. Feel free to coordinate a 4.1 release. We seem to be in a pretty stable period right now.